aten/src/ATen/native/native_functions.yaml

```diff
@@ -3016,6 +3035,11 @@
   variants: function
   supports_named_tensor: True
 
+- func: rsub.Scalar2(Scalar other, Tensor self, Scalar alpha=1) -> Tensor
```

Comment:
do we need rsub?  Why would I call this in C++?  Is it necessary for the python binding later?

Reply:
I don't think we need `rsub` (including the original ones), but we may need `rsub_`. I added merely because there are `rsub`s before. I can attempt to delete the existing `rsub`s if they are truly unecessary.